README.md

```diff
@@ -35,41 +35,37 @@ juju add-model core
 
 #### Dependencies
 
-Deploy the dependencies: [traefik](https://charmhub.io/traefik-k8s), [postgresql](https://charmhub.io/postgresql-k8s), [openfga](https://charmhub.io/openfga-k8s),
-and a certificates charm (e.g. [lego](https://charmhub.io/lego) or [self-signed-certificates](https://charmhub.io/self-signed-certificates)).
-And make sure they provide Juju offers:
+Deploy the dependencies: [traefik](https://charmhub.io/traefik-k8s), [postgresql](https://charmhub.io/postgresql-k8s),
+and a [certificates](https://charmhub.io/self-signed-certificates) charm using the `core` module:
 
 ```shell
-# Deploy dependencies
-juju deploy traefik-k8s traefik-public --trust --channel latest/stable
-juju deploy postgresql-k8s --trust --channel 14/stable
-juju deploy openfga-k8s --trust --channel latest/stable
-juju deploy self-signed-certificates --trust --channel latest/stable
-
-# Add integrations
-juju integrate openfga-k8s postgresql-k8s
-juju integrate traefik-public self-signed-certificates:send-ca-cert
-
-# Create the juju offers
-juju offer traefik-public:ingress ingress
-juju offer traefik-public:traefik-route traefik-route
-juju offer postgresql-k8s:database pg-database
-juju offer openfga-k8s:openfga openfga
-juju offer self-signed-certificates:send-ca-cert send-ca-cert
+terraform -chdir=modules/core init
+terraform -chdir=modules/core apply
 ```
 
-Because the bundle uses an external identity provider (e.g. Google or Microsoft Entra ID),
-it needs to provide additional variables for the module to run. More
-information about the IdP configuration can be
-found [here](https://charmhub.io/kratos-external-idp-integrator/configurations).
-Refer to [this](https://support.google.com/cloud/answer/15549257) article to find out how to create a private client in Google.
+#### Identity Platform
 
 Please create a Terraform variable definition (`.tfvars`) file in the root
 directory as follows:
 
 ```shell
 # vars.tfvars
-model = identity-platform
+postgresql_offer_url = "admin/core.pg-database"
+ingress_offer_url = "admin/core.ingress"
+send_ca_certificate_offer_url = "admin/core.send-ca-cert"
```

Comment:
I thought the plan was to have an `examples` dir that would bring all these modules together without requiring extra steps from the user.